access-control-app/src/cmr/access_control/services/acl_util.clj

```diff
@@ -105,6 +105,20 @@
                     identity-type-condition)]
     (get-acls-by-condition context condition)))
 
+(defn get-collections-chunked
+  "Searches for collections in chunks."
+  [context values keyword provider-id]
```

Comment:
We should avoid using clojure.core function names as bindings. Change keyword to field.